en/docs/streaming/migrating-from-stream-processor.md

```diff
@@ -0,0 +1,45 @@
+# Migrating from WSO2 Stream Processor
+
+The Streaming Integrator performs all functions that are also performed by [WSO2 Stream Processor](https://docs.wso2.com/display/SP440/Stream+Processor+Documentation). It also has additional features to trigger integration flows in order to take action in response to results derived after analyzing data.
+
+If you are currently using WSO2 Stream Processor to carry out any streaming integration/stream processing activities and want to carry them out in the Streaming Integrator, you can migrate your setup as follows:
+
+## Preparing to upgrade
+
+The following prerequisites should be completed before upgrading.
+
+- Make a backup of the SP 4.4.0 database and copy the <SP_HOME> directory in order to backup the product configurations.
+- Download the Streaming Integrator from the [Enterprise Integrator Home](https://wso2.com/integration/)
+
+!!! info "Updating the artifact IDs"
+    Note the following when migrating from WSO2 Streaming Processor(SP) to WSO2 Streaming Integrator(SI)
+    SP 4.x uses Siddhi 4.x and SI uses Siddhi 5.x from 1.x onwards. Therefore the group ID should be **renamed**
+    from `org.wso2.siddhi` to `io.siddhi`
+
+## Migrating Databases
+
+To connect the Streaming Integrator to the same databases as WSO2 SP 4.4.0 so that the persisted data can be accessed, configure the data sources as follows:
+
+- Configure the data sources in the `<SI_HOME>/conf/server/deployment.yaml` file the same way you have configured them in `<SP_HOME>/conf/wso2/worker/deployment.yaml` file.
+- Configure the data sources in the `<SI__TOOLING_HOME>/conf/server/deployment.yaml` file the same way you have configured them in `<SP_HOME>/conf/wso2/editor/deployment.yaml` file.
+- Check the data source configured for Business Rules  in the `<SP_HOME>/conf/wso2/dashboard/deployment.yaml` file, and configure that data source with the same parameter values in the `<SI__TOOLING_HOME>/conf/server/deployment.yaml` file.
```

Comment:
double spaces after "Business Rules"  

"Check the data source configured for Business Rules  in the ........"